app/jobs/finish_sync_job.rb

```diff
@@ -1,25 +1,29 @@
-class DataWarehouseSyncJob < ApplicationJob
-  def perform
-    return if nothing_to_sync?
+class FinishSyncJob < ApplicationJob
+  attr_accessor :sync
 
-    SlackBot.post('sync starting')
-    update_finished_imports
+  def perform(sync_id)
+    @sync = Sync.find_by id: sync_id
+    return unless sync
```

Comment:
I like this, but I'm not sure we're ready for it. We aren't logging anything at the moment - we just return from the job. I love that Rails provides this API and when we're ready, I will totally take you up on it! 😉